gateway-js/src/__tests__/gateway/executor.test.ts

```diff
@@ -1,6 +1,6 @@
 import gql from 'graphql-tag';
 import { ApolloGateway } from '../../';
-import { ApolloServer } from "apollo-server";
+import { ApolloServer } from 'apollo-server';
```

Comment:
sorry about the reformatting, it looks like jest did that when it inserted the "InlineSnapshot"